spec/views/users/show_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'users/show', type: :system do
+  it 'displays a user and their posts correctly' do
+    # Create a sample user with some associated posts
+    user = User.create(name: 'John Doe', email: 'john@example.com', password: 'password')
+    user.posts.create(title: 'Post 1', text: 'This is the first post')
+    user.posts.create(title: 'Post 2', text: 'This is the second post')
+
+    visit user_path(user)
+
+    # Expect user details to be displayed
+    expect(page).to have_content('John Doe')
+
+    # Expect user's posts to be displayed
+    expect(page).to have_content('Post 1')
+    expect(page).to have_content('This is the first post')
+    expect(page).to have_content('Post 2')
+    expect(page).to have_content('This is the second post')
+  end
```

Comment:
- [ ] Kindly make sure you ask what have been asked in the project requirements as you may see [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=corresponding%20wireframe%20view%3A-,Integration%20specs,-Use%20Capybara%20to) 
- [ ] You are supposed to use `Capybara` to do the integration test. You may consider checking [this article](https://www.codewithjason.com/rails-testing-hello-world-using-rspec-capybara/) if you have no idea about what to do.